solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -493,6 +493,11 @@ def f(x): return x  # NOQA
 
     data = []
     for metric_result in metrics:
+        # error here for ProbabilisticForecasts.
+        # metrics is a 2 element tuple:
+        # element 0: list of MetricsResults for each ProbFxConstVal in the ProbFx
+        # element 1: MetricsResult for distribution metrics (CRPS)
+        # so metric_result.values fails on the list object
```

Comment:
this isn't true anymore right? seems to me the calculator returns a flat list of MetricsResult

Reply:
correct. I added this comment before @awig changed the structure.

Reply:
Removed these comments